core/arch/arm/plat-imx/imx_caam.c

```diff
@@ -5,18 +5,58 @@
  * Bryan O'Donoghue <bryan.odonoghue@linaro.org>
  */
 
-#include <io.h>
+#include <imx.h>
 #include <imx_caam.h>
+#include <imx-regs.h>
+#include <io.h>
 #include <kernel/generic_boot.h>
 #include <platform_config.h>
 #include <stdint.h>
+#include <mm/core_memprot.h>
+
+
+static void caam_enable_clocks(bool enable)
+{
+	vaddr_t  ccm_base = core_mmu_get_va(CCM_BASE, MEM_AREA_IO_SEC);
+
+	uint32_t reg;
+	uint32_t mask;
```

Comment:
Please initialize

Reply:
Pushed a review commit with the initialized  variables.